tools/rum-models-generator/Package.swift

```diff
@@ -11,20 +11,45 @@ let package = Package(
         .package(name: "Difference", url: "https://github.com/krzysztofzablocki/Difference.git", from: "0.5.0"),
     ],
     targets: [
+        // CLI wrapper
         .target(
             name: "rum-models-generator",
             dependencies: [
-                "RUMModelsGeneratorCore",
+                "CodeGeneration",
+                "CodeDecoration",
                 .product(name: "ArgumentParser", package: "swift-argument-parser")
-            ]),
+            ]
+        ),
```

Comment:
As it is no longer `rum-` models generator, we may want to rename the package. I don't do this, because it has consequences on other files in the repo (all places where we call it from). I didn't want to spend time on it right now, but it's IMO something we might want consider in the future.

Reply:
Yes, it's fine keep it for now 👍